data/en/imagecaptcha.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "imageCaptcha",
+	"type": "function",
+	"syntax": "imageCaptcha()",
+	"returns": "any",
+	"related": ["imagecreatecaptcha"], ["imagewritetobrowser"],
+	"description": "Provides information about the time when the current application scope was created.",
+	"params": [
+            {"name":"text","description":"Text in the image captcha.","values":[],"default":"","type":"string","required":true},
+            {"name":"height","description":"Height of the image in pixels.","values":[""],"default":"","type":"numeric","required":true},
+            {"name":"width","description":"Width of the image in pixels.","values":[""],"default":"","type":"numeric","required":true},
```

Comment:
The `values` arrays here should be empty

Reply:
Thanks, error corrected.